docs/mise/tasks/lint-localization.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -euo pipefail
+
+cd "$MISE_PROJECT_ROOT"
+node scripts/lint-localization.mjs
```

Comment:
```suggestion
node -C $MISE_PROJECT_ROOT" scripts/lint-localization.mjs
```

nit: I'd think you should be able to do this, but what you have also is fine by me